cards/src/main/resources/cards/custom/group0/other_death_knight/oldgods/minion_general_vezax.json

```diff
@@ -0,0 +1,123 @@
+{
+  "name": "General Vezax",
+  "baseManaCost": 6,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 5,
+  "baseHp": 7,
+  "rarity": "LEGENDARY",
+  "description": "Taunt. Whenever this minion takes damage, switch between Windfury, Lifesteal, and Taunt.",
+  "trigger": {
```

Comment:
Use a `MODULO` operation, increment `RESERVED_INTEGER_2` on each damage received, and use three `AttributeAuras` with conditions instead. It'll be a lot less confusing. This will prevent the card from interfering with effects that permanently give it TAUNT or whatever, and it should still have copyable text.